playbooks/ci_latency_tests.yaml

```diff
@@ -23,3 +23,28 @@
         -d {{ sv_interface }} \
         -f "/tmp/ts_{{ inventory_hostname }}.txt"
       become: yes
+
+- name: Launch pcap sending
+  hosts:
+      - publisher_machines
+  become: true
+  tasks:
+    - name: "Create latency_tests directory if it doesn't exist"
+      file:
+        path: "{{ home }}/latency_tests/pcaps"
+        state: directory
+      tags: create_directory
+
+    - name: "Copy pcap file on publisher"
+      copy:
+        src: "{{ pcap_path }}/{{ pcap_file }}"
+        dest: "{{ home }}/latency_tests/pcaps"
+
+    - name: "Send pcap from publisher"
+      command: "bittwist -i {{ sv_interface }} \
+              {{ home }}/latency_tests/pcaps/{{ pcap_file }}"
+      register: cmd
+
+    - debug: msg= "{{ cmd.stderr }}"
```

Comment:
add a check to display debug only if ansible verbosity is set : `when: ansible_verbosity >= 2`